Compilation warnings

```
/paddle/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc: In instantiation of ‘void paddle::operators::FusionSeqExpandConcatFCOpKernel<T>::Compute(const paddle::framework::ExecutionContext&) const [with T = float]’:
/paddle/paddle/fluid/framework/op_registry.h:107:11:   required from ‘paddle::framework::OpKernelRegistrarFunctor<PlaceType, false, I, KernelTypes ...>::operator()(const char*, const char*) const::<lambda(const paddle::framework::ExecutionContext&)> [with PlaceType = paddle::platform::CPUPlace; long unsigned int I = 0ul; KernelTypes = {paddle::operators::FusionSeqExpandConcatFCOpKernel<float>, paddle::operators::FusionSeqExpandConcatFCOpKernel<double>}]’
/paddle/paddle/fluid/framework/op_registry.h:106:33:   required from ‘struct paddle::framework::OpKernelRegistrarFunctor<PlaceType, false, I, KernelTypes ...>::operator()(const char*, const char*) const [with PlaceType = paddle::platform::CPUPlace; long unsigned int I = 0ul; KernelTypes = {paddle::operators::FusionSeqExpandConcatFCOpKernel<float>, paddle::operators::FusionSeqExpandConcatFCOpKernel<double>}]::<lambda(const class paddle::framework::ExecutionContext&)>’
/paddle/paddle/fluid/framework/op_registry.h:105:38:   required from ‘void paddle::framework::OpKernelRegistrarFunctor<PlaceType, false, I, KernelTypes ...>::operator()(const char*, const char*) const [with PlaceType = paddle::platform::CPUPlace; long unsigned int I = 0ul; KernelTypes = {paddle::operators::FusionSeqExpandConcatFCOpKernel<float>, paddle::operators::FusionSeqExpandConcatFCOpKernel<double>}]’
/paddle/paddle/fluid/framework/op_registry.h:128:9:   required from ‘paddle::framework::OpKernelRegistrar<PlaceType, KernelType>::OpKernelRegistrar(const char*, const char*) [with PlaceType = paddle::platform::CPUPlace; KernelType = {paddle::operators::FusionSeqExpandConcatFCOpKernel<float>, paddle::operators::FusionSeqExpandConcatFCOpKernel<double>}]’
/paddle/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc:204:593:   required from here
/paddle/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc:139:74: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
/paddle/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc:141:66: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
/paddle/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc: In instantiation of ‘void paddle::operators::FusionSeqExpandConcatFCOpKernel<T>::Compute(const paddle::framework::ExecutionContext&) const [with T = double]’:
/paddle/paddle/fluid/framework/op_registry.h:107:11:   required from ‘paddle::framework::OpKernelRegistrarFunctor<PlaceType, false, I, KernelTypes ...>::operator()(const char*, const char*) const::<lambda(const paddle::framework::ExecutionContext&)> [with PlaceType = paddle::platform::CPUPlace; long unsigned int I = 1ul; KernelTypes = {paddle::operators::FusionSeqExpandConcatFCOpKernel<float>, paddle::operators::FusionSeqExpandConcatFCOpKernel<double>}]’
/paddle/paddle/fluid/framework/op_registry.h:106:33:   required from ‘struct paddle::framework::OpKernelRegistrarFunctor<PlaceType, false, I, KernelTypes ...>::operator()(const char*, const char*) const [with PlaceType = paddle::platform::CPUPlace; long unsigned int I = 1ul; KernelTypes = {paddle::operators::FusionSeqExpandConcatFCOpKernel<float>, paddle::operators::FusionSeqExpandConcatFCOpKernel<double>}]::<lambda(const class paddle::framework::ExecutionContext&)>’
/paddle/paddle/fluid/framework/op_registry.h:105:38:   required from ‘void paddle::framework::OpKernelRegistrarFunctor<PlaceType, false, I, KernelTypes ...>::operator()(const char*, const char*) const [with PlaceType = paddle::platform::CPUPlace; long unsigned int I = 1ul; KernelTypes = {paddle::operators::FusionSeqExpandConcatFCOpKernel<float>, paddle::operators::FusionSeqExpandConcatFCOpKernel<double>}]’
/paddle/paddle/fluid/framework/op_registry.h:112:9:   required from ‘void paddle::framework::OpKernelRegistrarFunctor<PlaceType, false, I, KernelTypes ...>::operator()(const char*, const char*) const [with PlaceType = paddle::platform::CPUPlace; long unsigned int I = 0ul; KernelTypes = {paddle::operators::FusionSeqExpandConcatFCOpKernel<float>, paddle::operators::FusionSeqExpandConcatFCOpKernel<double>}]’
/paddle/paddle/fluid/framework/op_registry.h:128:9:   required from ‘paddle::framework::OpKernelRegistrar<PlaceType, KernelType>::OpKernelRegistrar(const char*, const char*) [with PlaceType = paddle::platform::CPUPlace; KernelType = {paddle::operators::FusionSeqExpandConcatFCOpKernel<float>, paddle::operators::FusionSeqExpandConcatFCOpKernel<double>}]’
/paddle/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc:204:593:   required from here
/paddle/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc:139:74: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
/paddle/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc:141:66: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
```

--- a/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc
+++ b/paddle/fluid/operators/fusion_seqexpand_concat_fc_op.cc
@@ -136,9 +136,9 @@ class FusionSeqExpandConcatFCOpKernel : public framework::OpKernel<T> {
     // since infershape can not get lod info
     PADDLE_ENFORCE_EQ(ref_lod.size(), 1UL, "Only support input lod size is 1.");
     PADDLE_ENFORCE_EQ(in1_lod.size(), 1UL, "Only support input lod size is 1.");
-    PADDLE_ENFORCE_EQ(in1_lod[0].size() - 1, N,
+    PADDLE_ENFORCE_EQ(static_cast<int>(in1_lod[0].size() - 1), N,
                       "Batch size of all inputs should be equal.");
-    PADDLE_ENFORCE_EQ(in1_lod[0][N], N,
+    PADDLE_ENFORCE_EQ(static_cast<int>(in1_lod[0][N]), N,
                       "Seq_length of other inputs should be 1.");
     PADDLE_ENFORCE_EQ(in1_dims[0], N, "input height should be batch size.");
     for (size_t i = 2; i < ins.size(); ++i) {

--- a/paddle/fluid/operators/math/jit_kernel_test.cc
+++ b/paddle/fluid/operators/math/jit_kernel_test.cc
@@ -16,6 +16,7 @@ limitations under the License. */
 #include <sys/time.h>
 #include <cmath>    // for exp
 #include <cstring>  // for memcpy
+#include <random>
 #include <string>
 #include <vector>
 #include "gflags/gflags.h"

